lib/mercury/received_message.rb

```diff
@@ -38,14 +38,12 @@ def nack
     private
 
     def is_ackable
-      @work_queue_name != nil
+      !@work_queue_name.nil?
     end
 
     def performing_action(action)
-      is_ackable or raise "This message is not #{action}able"
-      if @action_taken
-        raise "This message was already #{@action_taken}ed"
-      end
+      is_ackable || raise("This message is not #{action}able")
+      raise "This message was already #{@action_taken}ed" if @action_taken
```

Comment:
Code A is more _readable_ than code B if it takes less time for an unacquainted developer to get the gist of code A. The gist is the typical path.

This is a bad transformation because it moves the unlikely path (`raise`) to a more prominent place at the beginning of the line, while shifting the important part (the decision on `@action_taken`) to the end of the line.